lib/SourceMapDevToolPlugin.js

```diff
@@ -284,6 +296,19 @@ class SourceMapDevToolPlugin {
 							] = new RawSource(sourceMapString);
 							chunk.files.push(sourceMapFile);
 						} else {
+							const dataForTemplateIdentifiers = {
+								chunk,
+								contentHash: createHash("md4")
+									.update(sourceMapString)
+									.digest("hex")
+							};
+							if (currentSourceMappingURLComment !== false) {
+								currentSourceMappingURLComment = compilation.getPath(
```

Comment:
Move the content of this `if` block after the next `if` block where `currentSourceMappingURLComment` can't be `false`.

`getPath` never returns false

Reply:
Yeah, getPath never returns false, but if i don't do assertion here, then it results in type error (maybe because compilation type was not declared for plugin)? Should i add type for compilation?
If i add type for compilation then this problem is solved but https://github.com/webpack/webpack/blob/master/lib/SourceMapDevToolPlugin.js#L105-L108 results in error, because it isn't of Tap type... should i ignore TS lint in this case? Or should make the other fields optional in Tap interface?